packages/jsdoc-plugins/lib/validator/types.js

```diff
@@ -35,6 +35,7 @@ const BASIC_TYPES = [
 	'ClientRect',
 
 	// Web APIs
```

Comment:
Window is part of DOM: https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model